sync/sync.go

```diff
@@ -223,8 +223,9 @@ func (sync *synchronizer) Fingerprint() string {
 		sync.state.LastBlockHeight())
 }
 
-func (sync *synchronizer) sendBlocksRequestIfWeAreBehind() {
-	if sync.peerSet.HasAnyValidSession() {
+func (sync *synchronizer) checkIfWeAreBehindTheNetwork() {
+	// TODO: write test for me
+	if sync.peerSet.HasAnyOpenSession() {
 		sync.logger.Debug("We have open seasson")
 		return
 	}
```

Comment:
I would suggest renaming this func to more descriptive one like `UpdateNodeHeight` or `UpdateNodeChain` since it actually requests blocks and joins topics

Reply:
`updateBlockchain` make more sense. I update the code